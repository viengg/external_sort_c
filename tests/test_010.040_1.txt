10
A\b3nRI:m$
20&Fk8E=%6
UhU4=d\[l[
%'ttCH5+.%
A7X6ln 'kO
^Vr8M?V}'D
*E'jhvQb@R
bu$F,Q<t{x
2#HJR-3sW3
u0J%q/+\<(
<NRi6|8d|4
A.)aA^ !{/
_\|gp`(`d=
vBEO;eoddV
>`&Yfh{][i
<KzsDKUaT6
XPPPp1pF1.
7_KqK:C"i:
rx_b]BT@SU
H{JFqm"L1a
z!nj8VtI8<
ps{na.V~Yu
Z2Wzv+\$KO
A<:q@KG/Rv
tcrca}RFHI
t$}qY[:+Q9
cgH)\d8m(!
<qkou$%waL
AK"<Nbsbw-
N@=bpYe,Ic
Z~6hV#?u,*
_rI2.Iq?Bj
*VO!+/LgbV
u{x*A_[V`m
(jaqU]H(i$
Q 67[u2}@k
c]n5:l,Vkx
1B<-d181bD
1cUGQvXD/G
}qB*\WOD6A
