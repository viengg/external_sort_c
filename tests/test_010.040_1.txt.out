10
%'ttCH5+.%
(jaqU]H(i$
*E'jhvQb@R
*VO!+/LgbV
1B<-d181bD
1cUGQvXD/G
2#HJR-3sW3
20&Fk8E=%6
7_KqK:C"i:
<KzsDKUaT6
<NRi6|8d|4
<qkou$%waL
>`&Yfh{][i
A.)aA^ !{/
A7X6ln 'kO
A<:q@KG/Rv
AK"<Nbsbw-
A\b3nRI:m$
H{JFqm"L1a
N@=bpYe,Ic
Q 67[u2}@k
UhU4=d\[l[
XPPPp1pF1.
Z2Wzv+\$KO
Z~6hV#?u,*
^Vr8M?V}'D
_\|gp`(`d=
_rI2.Iq?Bj
bu$F,Q<t{x
c]n5:l,Vkx
cgH)\d8m(!
ps{na.V~Yu
rx_b]BT@SU
t$}qY[:+Q9
tcrca}RFHI
u0J%q/+\<(
u{x*A_[V`m
vBEO;eoddV
z!nj8VtI8<
}qB*\WOD6A
